duckduckgo/Android
# Breakage form toast translations

Task/Issue URL: https://app.asana.com/0/0/1208820300773406/f

### Description

- Updated translations for toast that is displayed after a site breakage form is submitted via the privacy dashboard
- Updated privacy dashboard to latest development version

### Steps to test this PR

1. Submit a breakage report on a language other than English
2. Confirm that the toast that appears after submission is translated to the current language
